frontend/pages/shopping-lists/[id].vue

```diff
@@ -51,8 +53,10 @@
               />
             </v-col>
             <v-col
-              cols="6"
-              class="d-none d-lg-flex justify-center"
+              cols="12"
+              sm="4"
+              md="6"
```

Comment:
all of this can be done a lot simpler and the elements will be still in the same row. 
we can just add an `v-if="smAndUp" to the image column and add `    const { smAndUp } = useDisplay();` to the script and return it to the tempalte

```suggestion
              cols="12"
              sm="4"
              md="6"
```